reframe/frontend/ci.py

```diff
@@ -78,7 +78,7 @@ def rfm_command(testcase):
     return json
 
 
-def emit_pipeline(fp, testcases, child_pipeline_opts, backend='gitlab'):
+def emit_pipeline(fp, testcases, child_pipeline_opts=[], backend='gitlab'):
```

Comment:
It's never safe to have a mutable object as a default argument. I can fix that.

```suggestion
def emit_pipeline(fp, testcases, child_pipeline_opts=None, backend='gitlab'):
```